reference/functions/pgroonga-condition.md

```diff
@@ -84,23 +84,228 @@ Please refer to [Calling Functions][sql-syntax-calling-funcs] for information ab
 
 ## Usage
 
-## See also
+### Specify `index_name`
 
-* [postgres_fdw][postgres-fdw]
+シーケンシャルサーチ実行時でも、インデックスに指定したノーマライザーやトークナイザーのオプションを使って検索する方法を紹介します。
 
-* [normalizers_mapping][normalizers-mapping]
+`pgroonga_condition('keyword', index_name => 'pgroonga_index')`を使います。
+`index_name`にノーマライザーやトークナイザーを指定したインデックスの名前を指定します。
+
+Here are sample schema and data:
+
+```sql
+CREATE TABLE tags (
+  name text PRIMARY KEY
+);
+
+CREATE INDEX pgroonga_tag_name_index ON tags
+  USING pgroonga (name pgroonga_text_term_search_ops_v2)
+  WITH (normalizers='NormalizerNFKC150("remove_symbol", true)');
+
+INSERT INTO tags VALUES ('PostgreSQL');
+INSERT INTO tags VALUES ('Groonga');
+INSERT INTO tags VALUES ('PGroonga');
+INSERT INTO tags VALUES ('pglogical');
+```
+
+インデックスサーチ時はインデックスに指定したオプションを使ってインデックスサーチ時の挙動をカスタマイズできます。
+上のサンプルでは、`normalizers='...'`の部分でオプションを指定しています。
+
+一方、インデックスサーチではなくシーケンシャルサーチが実行されると、インデックスに指定されているオプションは参照できません。
+シーケンシャルサーチ時はどのインデックスを参照すればよいかという情報がないからです。
+
+そのためシーケンシャルサーチ実行時は、インデックスサーチ実行時と検索結果が異なる可能性があります。
+この問題を回避するためにシーケンシャルサーチ時に参照するインデックスを明示的に指定します。
+`pgroonga_condition()`の`index_name => '...'`がそのための引数です。
+
+次の例は、「`_p_G`」というキーワードで前方一致検索をしており、インデックスには`NormalizerNFKC150("remove_symbol", true)`が設定されています。
+[`remove_symbol`][remove-symbol]は記号を無視するオプションなので、「`_p_G`」は「`pg`」にノーマライズされます。
+（大文字が小文字になっているのは、`remove_symbol`オプションの挙動ではなく、`NormalizerNFKC150`のデフォルトの挙動によるものです。）
+そのため、このオプションが有効であれば、「`PGroonga`」と「`pglogical`」がヒットします。
+
+次の例は、シーケンシャルサーチですが、「`PGroonga`」と「`pglogical`」がヒットしていることが確認できます。
+このことから、シーケンシャルサーチ実行時でもインデックスに設定されている`NormalizerNFKC150("remove_symbol", true)`が参照できていることが確認できます。
```

Comment:
```suggestion

Next example execute a prefix search with the keyword "`_p_G`" while `NormalizerNFKC150("remove_symbol", true)` is configured to the index.   
Since [`remove_symbol`][remove-symbol] is the option to ignore the symbols, "`_p_G`" would be normalized to "`pg`".
( The reason why a capital letter `G` become a lower case latter `g` is not a behavior `remove_symbol` option, and is a default behavior of  `NormalizerNFKC150`.)
Therefore, "`PGroonga`" and "`pglogical`" should be hit while this option is valid. 

You can see "`PGroonga`" and "`pglogical`" are hit as result for the sequential search. 
This result shows that `NormalizerNFKC150("remove_symbol" configured to the index is referenced while the sequential search is executed.  

```

検索をしているというのを検索を実行しているという感じで`execute`にしていますが、「プログラムしている」みたいな感じのほうがいいのでしょうか・・・設定している？という感じでしょうか・・・。

Reply:
検索をしているというのを検索を実行しているという感じで大丈夫です。

Reply:
```suggestion
The next example executes a prefix search with the keyword "`_p_G`` while `NormalizerNFKC150("remove_symbol", true)` is specified in the index.
Since [`remove_symbol`][remove-symbol] is an option to ignore symbols, "`_p_G`" is normalized to "`pg`".
(The reason why the capital letter "`G`" becomes a lowercase "`g`" is not due to the `remove_symbol` option, but rather the default behavior of `NormalizerNFKC150`.)
Therefore, both "`PGroonga`" and "`pglogical`" should be hit while this option is active.
You can see that "`PGroonga`" and "`pglogical`" are hit as a result of the sequential search.
This result shows that `NormalizerNFKC150("remove_symbol" specified in the index is referenced even during sequential search execution.
```